File: src/main/kotlin/no/nav/arbeidsgiver/min_side/services/altinn/AltinnConfig.java
package no.nav.arbeidsgiver.min_side.services.altinn;

import lombok.Data;
import org.springframework.boot.context.properties.ConfigurationProperties;
import org.springframework.stereotype.Component;

@Component
@Data
@ConfigurationProperties(prefix = "altinn")
public class AltinnConfig {
    private String altinnHeader;
    private String altinnurl;
    private String APIGwHeader;
    private String proxyUrl;
    private String proxyFallbackUrl;
    private String proxyAudience;
}

File: src/main/kotlin/no/nav/arbeidsgiver/min_side/services/altinn/AltinnConfig.kt
package no.nav.arbeidsgiver.min_side.services.altinn

import org.springframework.boot.context.properties.ConfigurationProperties

@ConfigurationProperties(prefix = "altinn")
data class AltinnConfig(
    var altinnHeader: String = "",
    var altinnurl: String = "",
    var APIGwHeader: String = "",
    var APIGwUrl: String = "",
)